challenges-webapp/src/main/webapp/vue-app/engagement-center/initComponents.js

```diff
@@ -46,6 +49,9 @@ const components = {
   'challenge-program': challengeProgram,
   'engagement-center': EngagementCenter,
   'challenge-no-results': NoResultFound,
+  'programs': Programs,
+  'programs-list': ProgramsList,
+  'program-card': ProgramCard,
```

Comment:
Bad Vue component naming, it should be : 
```suggestion
  'engagement-center-programs': Programs,
  'engagement-center-programs-list': ProgramsList,
  'engagement-center-program-card': ProgramCard,
```

See [DEVSTD-34](https://github.com/Meeds-io/meeds/blob/develop/CODING_GUIDELINES.md#devstd-34) for more details